### PR TITLE
Add global nav, stock Vanilla navigation and first pass of homepage

### PR DESCRIPTION
**Pull request by deadlight**
## Done

- Included global-nav from NPM
- Tidied up the markup and simplified the homepage
- Used stock Vanilla navigation
- Styled the footer removing custom styles
- Removed all unused files

## QA

- `./run`
- Check that the homepage renders and works well in-browser

### CODE REVIEW

File: static/js/scratch.js [hunk 1]
```diff
@@ -16,172 +16,170 @@ var yuiOptions = {
...
+    var feedURL = "https://insights.ubuntu.com/feed/";
+    if (tag) {
+      feedURL += "?tag=" + tag;
+    }
+
+    if (limit) {
+      var feedLimit = limit;
+    } else {
+      var feedLimit = 5;
+    }
```
Comment (tbille) [inline static/js/scratch.js]: `feedLimit = limit | 5;`

File: package.json [hunk 2]
```diff
@@ -7,9 +7,10 @@
   },
   "devDependencies": {
     "node-sass": "^4.5.3",
-    "sass-lint": "^1.10.2"
+    "sass-lint": "^1.10.2",
+    "vanilla-framework": "1.8.x"
```
Comment (tbille) [inline package.json]: `vanilla-framework` as a `devDependencies`?

File: templates/index.html [hunk 3]
```diff
@@ -7,95 +7,73 @@
...
+<section class="p-strip is-bordered">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+        <h2 class="p-heading--three">Ubuntu is backed by Canonical</h2>
+        <p>Canonical is a software development and technology services company with staff in more than 30 countries around the world. Created alongside Ubuntu in 2004, it supports the platform’s development in many ways, from assisting with design and development to certification and the management of partner programmes.</p>
+        <p>We help you deliver your products without the cost and complexity associated with platform development. We provide training, support, QA and hands-on engineering expertise when you need it. Not to mention the marketing benefits of formalising your relationship with Ubuntu.</p>
+        <p><a href="/partnering-with-us">Learn more about Ubuntu and Canonical&nbsp;&rsaquo;</a></p>
+    </div>
+    <div class="col-4">
+        <img class="priority-0" width="200" height="200" alt="Canonical Logo" src="https://assets.ubuntu.com/v1/60d9b81e-picto-canonical.svg">
```
Comment (anthonydillon) [inline templates/index.html]: Replace `priority-0` with `u-hide--small`

File: static/js/scratch.js [hunk 4]
```diff
@@ -16,172 +16,146 @@ var yuiOptions = {
...
-core.setupHtmlClass();
-core.parallaxBackground();
-core.homeAnimation();
-core.svgFallback();
-core.autoLastItem();
+  core.setupAccordion();
+  core.cookiePolicy();
+  core.setupHtmlClass();
+  core.parallaxBackground();
+  core.autoLastItem();
```
Comment (anthonydillon) [inline static/js/scratch.js]: Let's lose all this js. Looks like the feeds are via the template.

File: templates/templates/footer.html [hunk 5]
```diff
@@ -1,55 +1,61 @@
...
+          <li class="p-list__item"><a href="/find-a-partner">Find a partner</a></li>
+          <li class="p-list__item"><a href="/partnering-with-us">Partnering with us</a></li>
+          <li class="p-list__item"><a href="/contact-us">Contact us</a></li>
+        </ul>
+      </div>
+      <div class="col-4">
+        <h2 class="p-muted-heading p-heading--four">Follow us</h2>
+        <ul class="p-inline-list u-no-margin--top">
+          <li class="p-inline-list__item">
+            <a href="https://twitter.com/Ubuntu" title="@ubuntu on Twitter"><i class="p-icon--facebook">@ubuntu on Twitter</i></a>
```
Comment (anthonydillon) [inline templates/templates/footer.html]: This should be `p-icon--twitter`

File: templates/templates/footer.html [hunk 6]
```diff
@@ -1,55 +1,61 @@
...
+          </li>
+          <li class="p-inline-list__item">
+            <a href="https://plus.google.com/+Ubuntu/posts" title="Ubuntu on Google+"><i class="p-icon--google">Ubuntu on Google+</i></a>
+          </li>
+          <li class="p-inline-list__item">
+            <a href="https://www.facebook.com/ubuntulinux?fref=ts" title="Ubuntu on Facebook"><i class="p-icon--twitter">Ubuntu on Facebook</i></a>
```
Comment (anthonydillon) [inline templates/templates/footer.html]: `p-icon--facebook`

File: templates/templates/footer.html [hunk 7]
```diff
@@ -1,55 +1,61 @@
...
+          </li>
+          <li class="p-inline-list__item">
+            <a href="http://blog.canonical.com" title="Canonical blog"><i class="p-icon--canonical">Canonical blog</i></a>
```
Comment (anthonydillon) [inline templates/templates/footer.html]: blog.ubuntu.com

File: templates/templates/footer.html [hunk 8]
```diff
@@ -1,55 +1,61 @@
...
+        </ul>
+      </div>
+      <div id="additional-info" class="col-3">
+        <h2 class="p-muted-heading p-heading--four">Find out more</h2>
+        <div>
+        <ul class="p-list">
+          <li class="p-list__item"><a href="https://www.ubuntu.com/">Ubuntu</a></li>
+          <li class="p-list__item"><a href="https://www.ubuntu.com/support/plans-and-pricing">Ubuntu Advantage</a></li>
+          <li class="p-list__item"><a href="https://www.canonical.com/">Canonical</a></li>
+          <li class="p-list__item"><a href="https://insights.ubuntu.com/">Press and resources</a></li>
```
Comment (anthonydillon) [inline templates/templates/footer.html]: Update link to canonical announcments on the blog.

File: static/js/scratch.js (hunk 4 above)
Comment (deadlight) [reply static/js/scratch.js]: RSS feeds are still loaded with JS on the partner pages

**Pull request merged**